src/privileged/FirefoxMonitor.jsm

```diff
@@ -328,10 +351,28 @@ this.FirefoxMonitor = {
     let win = doc.defaultView;
     let panelUI = doc.defaultView.FirefoxMonitorPanelUI;
 
+    let animatedOnce = false;
     let populatePanel = (event) => {
       switch (event) {
         case "showing":
           panelUI.refresh(this.domainMap.get(host));
+          if (animatedOnce) {
+            // If we've already animated once for this site, don't animate again.
+            doc.getElementById("notification-popup")
+               .setAttribute("fxmonitoranimationdone", "true");
+            doc.getElementById(`${this.kNotificationID}-notification-anchor`)
+               .setAttribute("fxmonitoranimationdone", "true");
+            break;
+          }
+          // Make sure we animate if we're coming from another tab that has
+          // this attribute set.
+          doc.getElementById("notification-popup")
+             .removeAttribute("fxmonitoranimationdone");
+          doc.getElementById(`${this.kNotificationID}-notification-anchor`)
+             .removeAttribute("fxmonitoranimationdone");
```

Comment:
nit: you *could* use

          doc.getElementById("notification-popup")
             .toggleAttribute("fxmonitoranimationdone", animatedOnce);

Since none of your code relies on the "true" value to be set. But I would leave that up to you...

Reply:
Ah, sorry, you probably can't, since `toggleAttribute` was introduced in 63, I think...

Anyway, I'll leave this here for posterity.